public/css/desktop.css

```diff
@@ -214,7 +184,7 @@ body {
 }
 
 .sortMenuButton:hover {
-	color: 1f268b;
+	color: #1f268b;
```

Comment:
Needed to add # before every hex color BECAUSE- forgot to declare <!DOCTYPE HTML> at top of html file until now and had to now because one thing I needed to work was not working(really stupid thing to forget, my bad). So I declared the doctype and then all the colors went all out of whack. I tried to find all color locations that would be wrong. If any colors do not look right on the page please let me know.

Changing doctype also made adding classes to html elements case sensitive (before adding DOCTYPE they were not case sensitive) and therefore some things may not look correct anymore. If some things are looking out of place or just plain bad, please let me know. 

DOCTYPE is something that I should have declared before writing anything in any file, and it was stupid of me to forget it. Declaring it now has caused some issues, but im pretty sure I have found them all.